app/controllers/concerns/accounts_controller.rb

```diff
@@ -0,0 +1,41 @@
+class AccountsController < ApplicationController
+  def authenticate
```

Comment:
This is all good, just note that it's common in Rails to see login/logout in a "Sessions Controller"